features/bevragen/persoon-beperkt/adressering/adres-regels/fields-alias.feature

```diff
@@ -0,0 +1,178 @@
+#language: nl
+
+Functionaliteit: Persoon beperkt: adressering velden vragen met fields/fields alias
+
+  Wanneer een consumer van de BRP API niet is geautoriseerd voor het bevragen van de adresregel velden voor een verblijfplaats buitenland
+  Dan moet de consumer de adresseringBinnenland fields alias gebruiken om aan te geven dat hij adresregel velden voor een verblijfplaats binnenland vraagt
+
+Rule: de standaard adresregel veld paden kan worden gebruikt door een consumer die is geautoriseerd voor het vragen van adresregels horende bij verblijfplaats binnenland (adres, locatie) en verblijfplaats buitenland
+
+  Abstract Scenario: consumer is geautoriseerd voor 'adressering buitenland' en vraagt <sub titel>
+    Gegeven de consumer is geautoriseerd voor 'adressering binnenland' gegevens
+    En de consumer is geautoriseerd voor 'adressering buitenland' gegevens
+    En de persoon met burgerservicenummer '000000097' heeft de volgende gegevens
+    | naam                  | waarde   |
+    | geslachtsnaam (02.40) | Maassen  |
+    | geboortedatum (03.10) | 19830526 |
+    En de persoon heeft de volgende 'verblijfplaats' gegevens
+    | naam                             | waarde                      |
+    | land adres buitenland (13.10)    | 6014                        |
+    | regel 1 adres buitenland (13.30) | 1600 Pennsylvania Avenue NW |
+    | regel 2 adres buitenland (13.40) | Washington, DC 20500        |
+    | regel 3 adres buitenland (13.50) | Selangor                    |
+    Als personen wordt gezocht met de volgende parameters
+    | naam          | waarde                              |
+    | type          | ZoekMetGeslachtsnaamEnGeboortedatum |
+    | geslachtsnaam | Maassen                             |
+    | geboortedatum | 1983-05-26                          |
+    | fields        | <fields>                            |
+    Dan heeft de response een persoon met de volgende 'adressering' gegevens
```

Comment:
Moet in de onderstaande tabel voor het eerste voorbeeld niet ook de velden 'aanhef', 'aanschrijfwijze' en 'gebruikInLopendeTekst' worden opgenomen?

Reply:
Functioneel heb je gelijk, maar bij zoeken (en niet raadplegen) is de eerste voorbeeld niet van toepassing. Wordt verwijderd